src/stores/sequencing.ts

```diff
@@ -12,13 +14,27 @@ import {
   type Parcel,
   type ParcelBundle,
   type ParcelToParameterDictionary,
+  type SequenceFilter,
   type UserSequence,
   type Workspace,
 } from '../types/sequencing';
 import effects from '../utilities/effects';
 import gql from '../utilities/gql';
 import { gqlSubscribable } from './subscribable';
 
+/* Environment Configuration */
+export const sequenceExpansionMode: Writable<SequencingMode> = writable(
+  env.PUBLIC_SEQUENCING_MODE === 'legacy' ? SequencingMode.LEGACY : SequencingMode.TEMPLATING,
```

Comment:
This sets `SequencingMode.LEGACY`, but all the UI elements check for `SequencingMode.EXPANSION`.

This fix go the other way, too -- but there are lots of uses of `SequencingMode.EXPANSION` and only one usage of `SequencingMode.LEGACY`. Not sure if there's one term we prefer.

```suggestion
  env.PUBLIC_SEQUENCING_MODE === 'legacy' ? SequencingMode.EXPANSION : SequencingMode.TEMPLATING,
```